Unmodified chart created by `helm create` doesn't pass `helm lint --strict`

If you run `helm create` to create a chart , then immediately try to run `helm lint --strict` on the created chart, it will report errors:

```
$ helm create testchart
Creating testchart
 $ helm lint testchart
==> Linting testchart
[INFO] Chart.yaml: icon is recommended

1 chart(s) linted, no failures
$ helm lint --strict testchart
==> Linting testchart
[INFO] Chart.yaml: icon is recommended
[ERROR] templates/: render error in "testchart/templates/service.yaml": template: testchart/templates/_helpers.tpl:15:14: executing "testchart.fullname" at <.Values.fullnameOver...>: map has no entry for key "fullnameOverride"

Error: 1 chart(s) linted, 1 chart(s) failed
```

This seems incorrect - one would assume that the default chart would be pedantically correct, even under the most strict lint check.

Output of `helm version`:
```
$ helm version -c
Client: &version.Version{SemVer:"v2.8.2", GitCommit:"a80231648a1473929271764b920a8e346f6de844", GitTreeState:"clean"}
```

Output of `kubectl version`: N/A

Cloud Provider/Platform (AKS, GKE, Minikube etc.):  N/A

--- a/pkg/chartutil/create.go
+++ b/pkg/chartutil/create.go
@@ -59,6 +59,9 @@ image:
   tag: stable
   pullPolicy: IfNotPresent
 
+nameOverride: ""
+fullnameOverride: ""
+
 service:
   type: ClusterIP
   port: 80

